controllers/rcj_soccer_referee_supervisor/referee/penalty_area_checker.py

```diff
@@ -0,0 +1,73 @@
+from typing import List
+
+from referee.consts import BLUE_PENALTY_AREA, YELLOW_PENALTY_AREA
+
+
+class PenaltyAreaChecker:
+    def __init__(self, allowed: int, reset_after: int, time: int):
```

Comment:
@Adman would it be better to call this `time_allowed` instead? I had a bit of a hard time parsing out what `allowed` meant.

Reply:
Good catch!